app/src/main/java/de/danoeh/antennapod/dialog/EpisodesApplyActionFragment.java

```diff
@@ -218,7 +216,11 @@ public void onStop() {
     }
 
     private void showSpeedDialIfAnyChecked() {
-        mSpeedDialView.setVisibility(checkedIds.size() > 0 ? View.VISIBLE : View.GONE);
+        if (checkedIds.size() > 0) {
+            mSpeedDialView.show();
```

Comment:
I think show() should only be called if it is not already shown. That function does some strange reflection stuff internally, could be a performance problem.

https://github.com/leinardi/FloatingActionButtonSpeedDial/blob/e6602c7bec57e534faaf5d9822b540a1ba1647f3/library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java#L192

Reply:
check added for just in case.

The reflection stuff is not applicable currently because the speeddial library used by AntennaPod is the older `v1.0.2`
https://github.com/leinardi/FloatingActionButtonSpeedDial/blob/1.0.2/library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

`1.0.2` was used due to support library compatibility concern:
AntennaPod used to use support v27.1.1 while FAB SpeedDial v2.x (and newer) uses support v28+